src/AutoUI/index.tsx

```diff
@@ -135,6 +154,11 @@ export const AutoUI = <T extends AutoUIBaseResource<T>>({
 	}, [modelRaw]);
 
 	const [filters, setFilters] = React.useState<JSONSchema[]>([]);
+	const [sort, setSort] = React.useState<TableSortOptions<T> | null>(null);
+	const [internalPagination, setInternalPagination] = React.useState<{
+		page: number;
+		itemsPerPage: number;
+	}>({ page: 0, itemsPerPage: pagination?.itemsPerPage ?? 50 });
```

Comment:
NIT: Let's get all `50`s of the repo to a single top level const (probably in the table.tsx).